File: sdkwork-ai-sdk-kotlin/src/main/kotlin/com/sdkwork/ai/TranscriptionResponse.kt
package com.sdkwork.ai

data class TranscriptionResponse(
    val text: String? = null,
    val task: String? = null,
    val language: String? = null,
    val duration: Double? = null,
    val words: List<Word>? = null,
    val segments: List<Segment>? = null
)

File: sdkwork-ai-sdk-csharp/Models/TranscriptionResponse.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class TranscriptionResponse
    {
        public string? Text { get; set; }
        public string? Task { get; set; }
        public string? Language { get; set; }
        public double? Duration { get; set; }
        public List<Word>? Words { get; set; }
        public List<Segment>? Segments { get; set; }
    }
}
